.jenkins

```diff
@@ -101,12 +101,13 @@ pipeline {
                     }
                     steps {
                         sh 'ccache --zero-stats'
-                        sh '''rm -rf build && mkdir -p build && cd build && \
+                        sh '''. /opt/intel/oneapi/setvars.sh --include-intel-llvm && \
+                              rm -rf build && mkdir -p build && cd build && \
                               cmake \
                                 -DCMAKE_BUILD_TYPE=Release \
                                 -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-                                -DCMAKE_CXX_COMPILER=clang++ \
-                                -DCMAKE_CXX_FLAGS="-fsycl-device-code-split=per_kernel -Werror -Wno-gnu-zero-variadic-macro-arguments -Wno-linker-warnings" \
+                                -DCMAKE_CXX_COMPILER=/opt/intel/oneapi/compiler/2023.0.0/linux/bin-llvm/clang++ \
```

Comment:
is not in the PATH?

Reply:
No, it wasn't, `icpx` would be but then it would use the wrong runtime. Locally, I have `clang++` in my path after calling `. /opt/intel/oneapi/setvars.sh --include-intel-llvm` but I don't know why that wasn't enough in the container. I stopped worrying about it once the tests compiled and ran successfully.

Reply:
Fine.  Might be worth exploring sourcing that file in the container entrypoint.
Not a priority though.

Out of curiosity I looked how NVIDIA does its containers entry points
https://gitlab.com/nvidia/container-images/cuda/-/blob/7d552bd45c5d76c9945f655950437672dc7c1f72/nvidia_entrypoint.sh
https://gitlab.com/nvidia/container-images/cuda/-/tree/master/entrypoint.d

Haven't looked whether they are images curated by Intel or Codeplay.